meth chocolate

Vaern opened the pull request:
Just what the doctor ordered!

Vaern committed:
added anvil recipes for waste barrels and recycling

Vaern committed:
Merge branch 'HbmMods:master' into master

Vaern committed:
an attempt to end the pain

prs make brain hurt

Vaern committed:
added meth chocolate and fixed fibrosis

Vaern commented:
bob please

HbmMods commented:
I'm stopping PRs and any major changes until UFFR has finished merging his PR with the main branch, that way I'll avoid creating even more conflicts. Just be patient.

Vaern committed:
Merge branch 'HbmMods:master' into master

Vaern committed:
ZIRNOX

still a WIP for now

Vaern committed:
Merge branch 'HbmMods:master' into master

Vaern committed:
fluid funnies

Vaern committed:
ZIRNOX balancing and changes

Vaern committed:
Final Balancing Changes

Vaern committed:
ZIRNOX basically done

Vaern committed:
final meth changes

meth choccy wasn't good enough for my liking

Vaern committed:
pain

extremely wip, hoping bob will accept PR so he can fix this mess

Vaern committed:
ZIRNOX meltdown

Vaern committed:
oops

Vaern committed:
stuff

Vaern committed:
Destroyed ZIRNOX

extremely wip, still need to finish the debris and fix the orientation for the ZIRNOX

Vaern committed:
more zirnox stuff

Vaern committed:
debris fix

Vaern committed:
pain

Vaern committed:
meltdown stuff

meltdowns are actually much more severe and therefore much more realistic

Vaern committed:
new ZIRNOX fuels

Vaern committed:
fix attempt

Vaern committed:
merge time

Vaern committed:
whoops, now ready for merging

Vaern committed:
had to readd recipe so duna probe isn't uncraftable

Once my PR is merged I might add a nuclear reactor crafting component for satellites so it can be deprecated for good

HbmMods commented:
Yeet
